res/css/views/rooms/_IRCLayout.scss

```diff
@@ -39,7 +39,7 @@ $irc-line-height: $font-18px;
             margin-right: $right-padding;
         }
 
-        > .mx_EventTile_msgOption {
+        .mx_EventTile_msgOption {
```

Comment:
Without `:not()` pseudo class, it is no longer required to increase specificity.